tests/ops/api/v1/endpoints/test_user_permission_endpoints.py

```diff
@@ -188,62 +189,123 @@ def test_edit_user_permissions(self, db, api_client, generate_auth_header) -> No
+        scopes = [USER_PERMISSION_READ]
+        auth_user = FidesUser.create(
             db=db,
-            data={"username": "user_1", "password": "test_password"},
+            data={"username": "auth_user", "password": "test_password"},
         )
-
-        permissions = FidesUserPermissions.create(
-            db=db, data={"user_id": user.id, "scopes": [PRIVACY_REQUEST_READ]}
+        ClientDetail.create_client_and_secret(
+            db,
+            CONFIG.security.oauth_client_id_length_bytes,
+            CONFIG.security.oauth_client_secret_length_bytes,
+            scopes=scopes,
+            user_id=auth_user.id,
         )
-        body = {"id": permissions.id, "scopes": [PRIVACY_REQUEST_READ]}
+        auth_header = generate_auth_header_for_user(auth_user, scopes)
+        invalid_user_id = "bogus_user_id"
+
         response = api_client.get(
             f"{V1_URL_PREFIX}/user/{invalid_user_id}/permission",
             headers=auth_header,
-            json=body,
         )
         permissions = FidesUserPermissions.get_by(
             db, field="user_id", value=invalid_user_id
         )
         assert HTTP_404_NOT_FOUND == response.status_code
         assert permissions is None
-        user.delete(db)
 
-    def test_get_user_permissions(self, db, api_client, generate_auth_header) -> None:
-        auth_header = generate_auth_header([USER_PERMISSION_READ])
-        user = FidesUser.create(
+    def test_get_user_permissions(self, db, api_client, user, permissions) -> None:
+        scopes = [USER_PERMISSION_READ]
+        auth_user = FidesUser.create(
             db=db,
-            data={"username": "user_1", "password": "test_password"},
+            data={"username": "auth_user", "password": "test_password"},
         )
-        permissions = FidesUserPermissions.create(
-            db=db, data={"user_id": user.id, "scopes": [PRIVACY_REQUEST_READ]}
+        ClientDetail.create_client_and_secret(
+            db,
+            CONFIG.security.oauth_client_id_length_bytes,
+            CONFIG.security.oauth_client_secret_length_bytes,
+            scopes=scopes,
+            user_id=auth_user.id,
         )
+        auth_header = generate_auth_header_for_user(auth_user, scopes)
 
         response = api_client.get(
-            f"{V1_URL_PREFIX}/user/{user.id}/permission", headers=auth_header
+            f"{V1_URL_PREFIX}/user/{user.id}/permission",
+            headers=auth_header,
         )
         response_body = json.loads(response.text)
         assert HTTP_200_OK == response.status_code
         assert response_body["id"] == permissions.id
         assert response_body["user_id"] == user.id
         assert response_body["scopes"] == [PRIVACY_REQUEST_READ]
-        user.delete(db)
+
+    def test_get_current_user_permissions(self, db, api_client) -> None:
+        # Note: Does not include USER_PERMISSION_READ.
+        scopes = [PRIVACY_REQUEST_READ, SAAS_CONFIG_READ]
+        auth_user = FidesUser.create(
```

Comment:
It looks like the `FidesUser.create` block is used 4 times to make the `auth_user`. This could potentially become a fixture that handles creating and deleting the `auth_user` for each test